ext/dcos-installer/dcos_installer/config/__init__.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, overrides={}, config_path=CONFIG_PATH, write_default_config=T
 ssh_port: 22
 process_timeout: 10000
 bootstrap_url: file:///opt/dcos_install_tmp
+ssh_parallelism: 20
```

Comment:
sounds good. We will default to 20 concurrent sessions but we will not add `ssh_parallelism` to a newly generated config.